pub-critic/src/app/components/FollowsTable/FollowsTable.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { User } from "common/interfaces";
+import { useState } from "react";
+import classes from "./FollowsTable.module.scss";
+import UserCard from "components/UserCard";
+import clsx from "clsx";
+
+interface FollowsTableProps {
+  followers: User[];
+  following: User[];
+  firstTab?: number;
+}
+
+export const FollowsTable = ({
+  followers,
+  following,
+  firstTab,
+}: FollowsTableProps) => {
+  const [tab, setTab] = useState<number>(firstTab || 0);
```

Comment:
You can use enum for tab names

Reply:
:white_check_mark:

Reply:
at the end becuase there were only two values it turned out to be a lot simpler to just use boolean